src/main.ts

```diff
@@ -124,28 +124,12 @@ function loadTscConfig(args: string[], allDiagnostics: ts.Diagnostic[]):
   return {options, fileNames};
 }
 
-export interface ClosureJSOptions {
-  tsickleCompilerHostOptions: tsickle.Options;
-  tsickleHost: tsickle.TsickleHost;
-  files: Map<string, string>;
-  tsicklePasses: tsickle.Pass[];
-}
-
-function getDefaultClosureJSOptions(fileNames: string[], settings: Settings): ClosureJSOptions {
+function getDefaultTsickleHost(fileNames: string[]): tsickle.TsickleHost {
```

Comment:
Is there a way to explain briefly what "sensible" means? The way the comment currently reads, it doesn't add all that much information on top of the method name. If it's too hard for you to figure out what this does, let's rather keep the comment shorter.